source/Ship.h

```diff
@@ -667,6 +669,9 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	Point acceleration;
 
 	int crew = 0;
+	// The preferred crew that the player has set, so we can re-hire that number even after landing on a planet without
+	// hiring service (PlayerInfo.desiredCrew was set from Crew() before, leading to a decreasing crew count).
```

Comment:
I think this is overly verbose.
```suggestion
	// hiring service.
```